packages/react-components/src/components/VideoTile.tsx

```diff
@@ -172,11 +172,9 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
 
   const ids = useIdentifiers();
 
-  const isSpeakingStyles = isSpeaking
-    ? {
-        border: `0.25rem solid ${theme.palette.themePrimary}`
-      }
-    : {};
+  const isSpeakingStyles = {
+    border: `0.25rem solid ${isSpeaking ? theme.palette.themePrimary : 'transparent'}`
```

Comment:
Please add a comment here describing why the transparent border is needed, so it's less magic a few months down the line.